lib/vquic/ngtcp2.c

```diff
@@ -1921,6 +1924,31 @@ static CURLcode do_sendmsg(size_t *psent, struct Curl_easy *data, int sockfd,
   else {
     assert(pktlen == (size_t)sent);
   }
+#else
+  ssize_t sent;
+  (void)qs;
+  (void)gsolen;
+
+  *psent = 0;
+
+  while((sent = send(sockfd, (const char *)pkt, pktlen, 0)) == -1 &&
```

Comment:
Is the cast necessary? Usually passing any pointer to a `void *` works just fine.

Reply:
Sadly it's necessary, because on Windows, [it is](https://docs.microsoft.com/windows/win32/api/winsock2/nf-winsock2-send) a `const char *`.

See also: https://github.com/curl/curl/issues/8978#issuecomment-1149997965